File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.cpp
class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) {
        vector<int> result;
        for(int i=0;i<nums.size();i++)
        {   int c = 0;
            for(int j =0 ; j<nums.size(); j++)
            {
                if(nums[j] < nums [i])
                    c++;
            }
            result.push_back(c);
        }
       return result;
    }
};

File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.py
class Solution(object):
    def smallerNumbersThanCurrent(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        result = []
        for i in nums:
            c = 0
            for j in nums:
                if j < i:
                    c += 1
            result.append(c)
        return result